go.mod

```diff
@@ -54,6 +54,7 @@ require (
 	github.com/opencontainers/selinux v1.8.5
 	github.com/pkg/errors v0.9.1
 	github.com/pmezard/go-difflib v1.0.0
+	github.com/prometheus/client_golang v1.11.2-0.20220215103819-989baa30fe95 // indirect
```

Comment:
why does this not use the exact v1.11.1 tag as your pr title states?

Reply:
err yes, I'll confirm and update the commit message. go get can't seem to resolve it to a valid tag for whatever reason.

Reply:
so `go get github.com/prometheus/client_golang@v1.11.1`  complains about it not being a valid tag. I manually updated go.mod and ran make vendor, which seems to work. PTAL.